lib/automation.go

```diff
@@ -365,7 +365,11 @@ func (automationImpl) Run(scope scope, p *RunParams) (string, error) {
 		}
 		p.WorkflowID = wf.WorkflowID()
 	}
-	return scope.AutomationOrchestrator().RunWorkflow(scope.AppContext(), workflow.ID(p.WorkflowID), p.RunID)
+	if p.RunID == "" {
```

Comment:
I feel like I'm blanking on a conversation we've already head. Shouldn't we _always_ be creating a new RunID here? What if I supply my own (bad) run ID?

Reply:
ok so either drop this on this PR, or add a TODO